examples/libh2o/simple.c

```diff
@@ -215,9 +215,6 @@ static int setup_ssl(const char *cert_file, const char *key_file, const char *ci
     }
 
 /* setup protocol negotiation methods */
-#if H2O_USE_NPN
-    h2o_ssl_register_npn_protocols(accept_ctx.ssl_ctx, h2o_http2_npn_protocols);
-#endif
 #if H2O_USE_ALPN
```

Comment:
Does it make sense to also use the ALPN define? That is, we would always use ALPN?

Reply:
@deweerdt Thank you for the review!
Practically, yes, we always use ALPN, as at lease major browsers do not support cleartext HTTP/2.
The current code base, however, supports OpenSSL < 1.0.2 so we can not assume that ALPN support is always there. Removing support for older OpenSSL would be worth to consider, though.

Reply:
> The current code base, however, supports OpenSSL < 1.0.2 so we can not assume that ALPN support is always there. Removing support for older OpenSSL would be worth considering, though.

that's a good point, @kazuho what do you think? Should we drop < 1.0.2 support?